collectors/utils.py

```diff
@@ -111,11 +111,15 @@ def handle_urls(references: list, source_ref: str) -> list:
 }
 
 
-def convert_cvss_score_to_impact(score: Union[Decimal, float]) -> Impact:
+def convert_cvss_score_to_impact(score: Decimal) -> Impact:
```

Comment:
This implies that we alwasy get a `Decimal` yet there have logic in case it's not?
If the score can be passed as a float I think it should be reflected here, even if later on it's all converted to a `Decimal`

Reply:
You are right, I overlooked this when moving the condition. Thanks!